docs/writing-policies/metadata.md

```diff
@@ -1,29 +1,32 @@
 ---
-sidebar_label: "Policy metadata"
-title: "Policy metadata"
+sidebar_label: Policy metadata
+title: Policy metadata
+description: Using policy metadata when developing a Kubewarden policy.
+keywords: [kubewraden, kubernetes policy development, policy metadata]
+doc-type: [how-to, tutorial]
+doc-topic: [kubewarden, writing-polices, policy-metadata]
+doc-persona: [kubewarden-developer]
 ---
 
-# Policy metadata
+The Kubewarden metadata file, `metadata.yaml`,
+is a configuration file containing important information and settings
+related to the policies used within Kubewarden.
+This documentation is overview of the purpose and usage of the metadata file.
 
-The Kubewarden metadata file, `metadata.yaml`, serves as a vital configuration
-file that contains important information and settings related to the policies
-used within Kubewarden. This documentation provides an overview of the
-purpose and usage of the metadata file.
+The policy `metadata.yaml` file has defaults for the policy,
+as well as metadata such as author and description,
+set by the policy author.
+The `kwctl annotate` command uses the file to annotate the `.wasm` file containing the policy.
+Therefore, all the relevant information required to run the policy is available.
+More information about how to annotate the policy is in the
+[Distributing Policies](../distributing-policies.md) guide.
 
-The policy `metadata.yaml` file contains defaults for the policy, in addition
-to metadata such as author and description, set by the policy author. The file
-is used by the `kwctl annonate` command to, as the name suggests, annotates the
-`.wasm` file containing the policy. Therefore, all the relevant information required to run
-the policy will be available. More information about how to annotate the policy
-can be found in the [Distributing Policies](../distributing-policies.md) guide.
-
-When policy users want to use a policy, they generate a YAML manifest using
-`kwctl scaffold`. This command reads the policy metadata embedded in the
-shipped Wasm module, performs sanity checks, and returns a YAML manifest that
-the author can use as-is or adapt to their needs.
+When policy users want to use a policy, they generate a YAML manifest using `kwctl scaffold`.
+This command reads the policy metadata embedded in the shipped Wasm module,
+performs checks, and returns a YAML manifest that the author can use as-is or modify.
 
 As a policy author, you can adapt the `metadata.yaml` file provided during the
-scaffolding of your policy.
+construction of your policy.
```

Comment:
```suggestion
scaffolding of your policy.
```

I would keep scaffolding, from the `kwctl scaffold` of last paragraph. If not, it may sound like a compiling step, which it isn't.